smart-contracts-hardhat/scripts/deployUniversalProfiileWithLSP23.ts

```diff
@@ -0,0 +1,149 @@
+// libs
+import { ethers } from 'hardhat';
+import { AbiCoder } from 'ethers';
+import { ERC725 } from '@erc725/erc725.js';
+
+// LSPs artifacts
+import LSP23FactoryArtifact from '@lukso/lsp-smart-contracts/artifacts/LSP23LinkedContractsFactory.json';
+import UniversalProfileInitArtifact from '@lukso/lsp-smart-contracts/artifacts/UniversalProfileInit.json';
+
+// ERC725.js schemas
+import LSP1UniversalReceiverDelegateSchemas from '@erc725/erc725.js/schemas/LSP1UniversalReceiverDelegate.json';
+import LSP3ProfileMetadataSchemas from '@erc725/erc725.js/schemas/LSP3ProfileMetadata.json';
+import LSP6KeyManagerSchemas from '@erc725/erc725.js/schemas/LSP6KeyManager.json';
+
+const LSP23_FACTORY_ADDRESS = '0x2300000A84D25dF63081feAa37ba6b62C4c89a30';
+const LSP23_POST_DEPLOYMENT_MODULE = '0x000000000066093407b6704B89793beFfD0D8F00';
```

Comment:
Suggest adding `_ADDRESS` for consistency. 

```suggestion
const LSP23_POST_DEPLOYMENT_MODULE_ADDRESS = '0x000000000066093407b6704B89793beFfD0D8F00';
```

It will also read better in the code below:

<img width="759" alt="image" src="https://github.com/lukso-network/lukso-playground/assets/31145285/84614bef-4b2a-48b7-b8dc-38c776233c21">